src/app.test.js

```diff
@@ -139,6 +123,26 @@ describe('Test sending events', () => {
     expect(numTimesVerifyAuthCalled[0]).toBe(1)
   })
 
+  test('calling event with caching behavior', async () => {
+    mockSuccessfulMixpanelEventTrackCall()
+    mockEnabledUserSamAuthCall()
+
+    const response1 = await request(app).post('/api/event')
+      .send({ event: 'foo', properties: { appId: 'test' } })
+      .set('Authorization', 'Bearer mysupersecrettoken')
+    expect(response1.statusCode).toBe(200)
+
+    expect(numTimesVerifyAuthCalled[0]).toBe(1)
```

Comment:
numTimesVerifyAuthCalled tracks how many times the http request to sam was actually called